requirement.txt

```diff
@@ -0,0 +1,15 @@
+argparse
```

Comment:
is this what pip freeze/conda gave? Might want to add versions too

Reply:
Yeah, but my version is old. I removed the version to make it compatible with any python and it worked.